services/src/atdd/src/test/java/org/openkilda/atdd/TopologyEventsBasicTest.java

```diff
@@ -118,8 +118,8 @@ public void linkIsAddedInTheMiddle() throws Exception {
         List<LinkDto> links = LinksUtils.dumpLinks();
         LinkDto middleLink = getMiddleLink(links);
 
-        String srcSwitch = getSwitchName(middleLink.getPath().get(0).getSwitchId());
-        String dstSwitch = getSwitchName(middleLink.getPath().get(1).getSwitchId());
+        String srcSwitch = getSwitchName(middleLink.getPath().get(0).getSwitchId().toString());
```

Comment:
Each SwitchId.toString() call is a reason to try to change argument type of called method. No more comments regarding such cases.